travis/build-and-test-downstream.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+
+build_order=(opm-common opm-parser opm-material opm-core opm-output opm-grid opm-simulators)
+
+# This shell script should be started with the name of a module as
+# only only command line argument, it will then build and test all the
+# modules downstream of this module. 
+#
+# Before invoking this script all the modules should have been cloned
+# in sibling directories, so that this script will see this directory
+# structure:
+#
+#   opm-common/
+#   opm-parser/
+#   opm-material/
+#   opm-output/
+#   opm-core/
+#   opm-grid/
+#   opm-simulators/
+#
+# This can typically be achived by using the 'clone-opm.sh' script.
+
+
+function build_and_test {
+    project=${1}
+    echo "Building: ${project}"
+    mkdir ${project}/build
```

Comment:
`mkdir -p` will not fail if build directory already exists.  This is convenient if you manually want to invoke this script (possibly several times, as I'm doing now).